### DISCUSSION TITLE
custom patches

### DISCUSSION THREAD

**Discussion by bennet-esyoil** [postalserver/postal#1575] [Development discussions]
Hey together, 

we are using postal on 2 Servers which have different purposes. Since there was no way at the time to disable suppression lists we implemented a custom patch that allowed us to disable them. 
This in itself is not that big of a deal, but after the upgrade we are not able to apply them again, since now everything lives in containers directly downloaded.

What would be the main way to go here? Build custom containers or share the patches in the main-repo to maybe have it merged into the main-branch?

Best Regards, Bennet

**Comment by willpower232** [comment]
You would have to create your own docker images to include custom code.

In this case, I am not sure your changes would be accepted as suppression lists are a good feature.

**Reply by bennet-esyoil** [reply]
In short: I agree with that. 

Long version: We send out up to 1200 Emails per day to single customers who rely on the delivery. So far postal has done a great job without any major outages or anything. Only problem for us are the suppression lists, since they would stop sending to these addresses for 1 day. Even 1 day is too much so we have to manually intervene there, which is made harder by the fact that there is no UI to clear addresses from the list.

I think making suppression-lists optional (aka. being enabled by default and disableable via config) would be one way to solve this problem for us. Other options could either be to implement shorter time-periods (i.E. 1 hour, 6 hours, etc.) or have removal of entries in the UI. 

If you are willing to guide me in a direction of which option would work with the postal-team I am willing to submit a PR for that purpose.
